chapter_introduction/index_vn.md

```diff
@@ -2005,7 +2011,9 @@ with initially unknown rewards, this problem
 is the classic *multi-armed bandit problem*.
 -->
 
-*dịch đoạn phía trên*
+Khi ở môi trường được quan sát đầy đủ, ta gọi vấn đề học tăng cường là *(Quy trình Quyết định của Markov - Markov Decision Process* (MDP).
```

Comment:
```suggestion
Khi ở môi trường được quan sát đầy đủ, ta gọi vấn đề học tăng cường là *Quá trình Quyết định Markov* (*Markov Decision Process* -- MDP).
```

Từ Process này được sử dụng trong "random process" -- được dịch là "quá trình ngẫu nhiên" trong nhiều tài liệu.

Reply:
`--` này được sử dụng như thế nào vậy anh nhỉ? nó có ý nghĩa như thế nào trong đoạn văn vậy ạ?

Reply:
Đó là dấu gạch dài thì phải.
https://stackoverflow.com/questions/52371734/how-to-make-a-long-in-markdown-i-e-the-character